lib/plugins/aws/package/compile/events/kafka.js

```diff
@@ -112,6 +124,37 @@ class AwsCompileKafkaEvents {
       functionObj.events.forEach((event) => {
         if (!event.kafka) return;
 
+        if (!event.kafka.accessConfigurations || event.kafka.accessConfigurations.length === 0) {
+          throw new ServerlessError(
+            `You must provide at least one accessConfiguration for function: ${functionName}`,
+            'FUNCTION_KAFKA_ACCESS_CONFIGURATION_EMPTY'
+          );
+        }
+
+        const {
+          accessConfigurations: {
+            vpcSecurityGroup,
+            vpcSubnet,
+            clientCertificateTlsAuth,
+            serverRootCaCertificate,
+          },
+        } = event.kafka;
+
+        if ((vpcSecurityGroup && !vpcSubnet) || (vpcSubnet && !vpcSecurityGroup)) {
+          const missing = vpcSecurityGroup ? 'vpcSubnet' : 'vpcSecurityGroup';
+          throw new ServerlessError(
+            `You must specify at least one ${missing} accessConfiguration for function: ${functionName}`,
+            `FUNCTION_KAFKA_VPC_ACCESS_CONFIGURATION_MISSING_${missing.toUpperCase()}`
+          );
+        }
+
+        if (serverRootCaCertificate && !clientCertificateTlsAuth) {
+          throw new ServerlessError(
+            `You cannot specify serverRootCaCertificate accessConfiguration without providing a clientCertificateTlsAuth accessConfiguration for function: ${functionName}`,
```

Comment:
let's put `""` around `serverRootCaCertificate` and `clientCertificateTlsAuth` to highlight it here

Reply:
good idea 👍